CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,6 @@
 # Community Code of Conduct
 
 Keycloak follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+## Reporting
+For incidents occurring in the Keycloak community, contact the Keycloak Code of Conduct Committee via `keycloak-coc@googlegroups.com`.
```

Comment:
I find "incidents" a little too vague. How about "If you suspect violations of the code of conduct in the Keycloak community,..."?